uber/templates/regform.html

```diff
@@ -97,6 +97,12 @@
         <label for="full_name" class="col-sm-2 control-label">Legal Name</label>
         <div class="col-sm-6">{{ attendee.full_name }}</div>
     </div>
+    <div class="form-group">
+        <label for="badge_type" class="col-sm-2 control-label">Badge Type</label>
+        <div class="col-sm-6">
+            {{ attendee.badge_type_label }}
+        </div>
+    </div>
```

Comment:
@EliAndrewC it shouldn't a line just above that is `{% if c.PAGE == 'confirm' %}`